lib/utils.js

```diff
@@ -463,6 +463,23 @@ function transformEachsCallbackArgs(callBack) {
   return callBack;
 }
 
+/**
+ * Determines if the argument to `get()` is positive or negative.
+ * @param {ASTNode} indexArg
+ * @returns {number} Defaults to 0
+ */
+function determineGetIndex(indexArg) {
+  const indexNode = indexArg[0];
+  let index = 0; // default value
+
+  if (indexNode && indexNode.value) {
+    index = indexNode.value;
+  } else if (indexNode && indexNode.operator && indexNode.argument && indexNode.argument.value) {
+    index = `${indexNode.operator}${indexNode.argument.value}`;
```

Comment:
This would return a string, while the function is specified to return a number. Maybe apply `parseInt()` to it?

Reply:
I tried that originally, but jscodeshift throws an error if it's a negative integer and I wasn't able to figure out why.

I'll update the jsdoc